### PR TITLE
add(flow cell exists check)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

Add a flow cell exists in statusdb check before trying to add it in new flow cell finish flow

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by ChrOertlin** [comment]
tested in stage:
image

**Review by diitaz93 — APPROVED** [review]
👍

**Comment by ChrOertlin** [comment]
deployed to stage and prod:
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/christian.oertlin
done.
Log deploy... done.
cg, version 37.0.15

File: cg/meta/demultiplex/demux_post_processing.py [hunk 1]
```diff
@@ -302,14 +302,18 @@ def file_exists_in_latest_version_for_bundle(
 
     def store_flow_cell_in_status_db(self, parsed_flow_cell: FlowCellDirectoryData) -> None:
```
Comment (seallard) [inline cg/meta/demultiplex/demux_post_processing.py:303]: I think this function name is a bit misleading now, we can fix it later

**Pull request merged**